plugin/user_plugin_server.py

```
@@ -116,6 +118,214 @@ async def plugin_status(plugin_id: Optional[str] = Query(default=None)):
+            except Exception as e:
+                logger.error(f"Error in lifecycle.startup: {e}")
+
+        # ========== 3. 定时任务：timer ==========
+        def _run_timer_interval(fn, interval_seconds: int, fn_name: str):
+            while True:
+                try:
+                    if asyncio.iscoroutinefunction(fn):
+                        asyncio.run(fn())
+                    else:
+                        fn()
+                except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)